packages/salesforcedx-lightning-lsp-common/src/__tests__/testUtils.ts

```diff
@@ -79,40 +83,37 @@ const readFileContent = (filePath: string): string => {
   return '';
 };
 
+// Helper function to recursively find all files in a directory
+const findFilesRecursively = (dirPath: string, basePath: string, files: Record<string, string>): void => {
+  try {
+    if (!fs.existsSync(dirPath)) {
+      return;
+    }
+    const entries = fs.readdirSync(dirPath, { withFileTypes: true });
+    for (const entry of entries) {
+      const fullPath = join(dirPath, entry.name);
+      if (entry.isDirectory()) {
+        findFilesRecursively(fullPath, basePath, files);
+      } else if (entry.isFile()) {
+        const relativePath = fullPath.substring(basePath.length + 1);
+        const content = readFileContent(fullPath);
+        if (content) {
+          files[relativePath] = content;
+        }
+      }
+    }
+  } catch {
+    // Ignore errors when reading directories
+  }
+};
```

Comment:
I haven't thought deeply about it, but what do you think is missing from readdir with `recursive` option (with file types), then filter out the dirs and map that to get content and build a giant object out of?

fine if you don't like it, since it's test code, but I'd like to get us in the habit of using all the current node features that people used to have to do work like this for